Makefile

```diff
@@ -294,6 +295,35 @@ txsim-build-docker:
 	docker build -t ghcr.io/celestiaorg/txsim -f docker/txsim/Dockerfile  .
 .PHONY: txsim-build-docker
 
+VERSION := $(shell git describe --tags --dirty --always)
+COMMIT  := $(shell git rev-parse HEAD)
+
+# build the -ldflags string here
+LLDFLAGS := \
+  -X github.com/cosmos/cosmos-sdk/version.Name=celestia-app \
+  -X github.com/cosmos/cosmos-sdk/version.AppName=celestia-appd \
+  -X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
+  -X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
+  -X github.com/celestiaorg/celestia-app/v4/cmd/celestia-appd/cmd.v2UpgradeHeight=0
+
+build-talis-bins:
```

Comment:
[blocking] need a doc that shows up in `make help` output

Reply:
f363f9e47cd9fa0b94cd04c020ff25b8d1947d46